app/controllers/user_controller.rb

```diff
@@ -0,0 +1,14 @@
+class UserController < ApplicationController
+  def email_preferences
+    redirect_to root_path unless can?(:update_email_preferences, current_user)
+    @selected = current_user.email_preference
```

Comment:
I was a little confused later in use of the word "selected", wonder if maybe @current_preference, selected makes me think a choice just happened in the gui.

Reply:
done